onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -202,16 +202,10 @@ CustomOpLibrary::CustomOpLibrary(const char* library_path, OrtSessionOptions& or
   {
     OrtPybindThrowIfError(platform_env.LoadDynamicLibrary(library_path, &library_handle_));
 
-    if (!library_handle_)
```

Comment:
These 2 checks are superfluous - `OrtPybinfThrowIfError()` called on the previous lines does the same thing, so removing these